### ISSUE TITLE
Docs show expanded contacts API but PR#210 hasn't been included in release 18

### ISSUE THREAD

**Issue by raarts** [expo/expo#301]
Hi, I just upgraded to 18.0.4, following the 18.0.0 blog procedure.

I don't get thumbnails back, even if I request them using Expo.Contacts.THUMBNAIL.
The returned object *does* say imageAvailable: true. (Tried this on iOS).
How do I retrieve the image?
Example code:

```
result = await Expo.Contacts.getContactsAsync({
  fields: [
    Expo.Contacts.PHONE_NUMBERS,
    Expo.Contacts.EMAILS,
    Expo.Contacts.THUMBNAIL,
    Expo.Contacts.URLS,
    Expo.Contacts.IMAGE,
  ],
  pageOffset: newOffset,
  pageSize: PAGESIZE,
})
```

Example returned Object:

```
company: "My Company"
contactType: "person"
department: ""
emails: Array(0)
firstName: "John"
id: "9EAFB58F-6D26-4043-B921-BAD670B167BC"
imageAvailable: true
jobTitle: ""
lastName: "Doe"
middleName: ""
name: "John Doe"
nickname: ""
phoneNumbers: Array(4)
previousLastName: ""
__proto__: Object
```

**Comment by raarts** [comment]
Looking into it some more reveals that this PR https://github.com/expo/expo/pull/210 has not been merged yet, even the the doc assumes it has been.

**Comment by terribleben** [comment]
It did get merged (we use a monorepo so the PR was closed by a bot): https://github.com/expo/expo/commit/8c9c42abe1bc3bae80d751156edfc0a52046e0c7

@aalices can you provide any more info about the original question (thumbnails)?

**Comment by aalices** [comment]
@terribleben I think the JS SDK has not bee merged. Code on GitHub hasn't got the changes. This is also the reason why @raarts has no `urls` in the response (as long as the contact has some).

**Comment by nikki93** [comment]
Should be fixed in 18.0.5, let me know if this issue still occurs.

**Comment by raarts** [comment]
Fixed! Thanks!
